proposed/http-client/http-client.md

```diff
@@ -17,7 +17,7 @@ It SHOULD implement exceptions for `Psr\Http\Client\Exception\NetworkException`
 ## Goal
 
 The goal of this PSR is to allow developers to create libraries decoupled from HTTP Client
-implementations. This would make libraries more stable since the reduced number of
+implementations. This would make libraries more stable since the number of
 dependencies and the likelihood to get in version conflicts is reduced.
```

Comment:
Before: 

> This would make libraries more stable since the **reduced** number of dependencies and the likelihood to get in version conflicts is reduced.

After:

> This would make libraries more stable since the number of dependencies and the likelihood to get in version conflicts is reduced.